.travis.yml

```diff
@@ -20,3 +20,15 @@ script:
     - inspekt lint
     - inspekt indent
     - inspekt style
+    - |
+        err=false
+        for file in $(git diff --name-only $TRAVIS_COMMIT_RANGE -- *.py)
+        do
+            echo -n Checking license in file $file...
+            inspekt license --copyright "Copyright" --author "Author" $file || err=true
```

Comment:
Nice, there is only one problem I see, inspekt only supports GPLv2+ license for now (even though it claims GPLv2). And we have GPLv2_ONLY code getting to avocado and others like LGPL might follow.